### PR TITLE
[Nordigen] Can now select multiple scopes

### PR DESCRIPTION
**Pull request by LucBerge**
Possibility to select multiple scopes when creating a requisition link

### CODE REVIEW

**Comment by pipedream-component-development** [comment]
Thank you so much for submitting this! We've added it to our backlog to review, and our team has been notified.

**Comment by pipedream-component-development** [comment]
Thanks for submitting this PR! When we review PRs, we follow the [Pipedream component guidelines](https://pipedream.com/docs/components/guidelines/). If you're not familiar, here's a quick checklist:
- [ ] [Create components to address specific use cases whenever possible](https://pipedream.com/docs/components/guidelines/#component-scope)
- [ ] [Component `key`s should follow the format `app_name_slug-slugified-component-name`](https://pipedream.com/docs/components/guidelines/#component-key-pattern)
- [ ] Components should follow the [standard directory structure](https://pipedream.com/docs/components/guidelines/#folder-structure)
- [ ] [Prefer Node.js client libraries to REST APIs](https://pipedream.com/docs/components/guidelines/#using-apis-vs-client-libraries)
- [ ] When making API requests, [handle pagination](https://pipedream.com/docs/components/guidelines/#pagination) to ensure all data / events are processed
- [ ] [Use `secret` props](https://pipedream.com/docs/components/guidelines/#capturing-sensitive-data) to capture sensitive data
- [ ] Props and methods should be defined in [app files](https://pipedream.com/docs/components/guidelines/#app-files) whenever possible
- [ ] Document methods with [JS Docs](https://pipedream.com/docs/components/guidelines/#methods)
- [ ] [Use `optional` props](https://pipedream.com/docs/components/guidelines/#optional-vs-required-props) whenever possible, and [set a `default` value](https://pipedream.com/docs/components/guidelines/#default-values) where you can
- [ ] Use [async options](https://pipedream.com/docs/components/guidelines/#async-options) to accept user input wherever possible

File: components/nordigen/actions/create-requisition-link/create-requisition-link.mjs [hunk 1]
```diff
@@ -4,7 +4,7 @@ export default {
   key: "nordigen-create-requisition-link",
   name: "Create Requisition Link",
   description: "Create a requisition link and id to be used in other Nordigen actions. [See the docs](https://nordigen.com/en/account_information_documenation/integration/quickstart_guide/)",
-  version: "0.0.1",
+  version: "0.0.2",
```
Comment (dylburger) [inline components/nordigen/actions/create-requisition-link/create-requisition-link.mjs:7]:
```suggestion
  version: "0.0.2",
```

Comment (dylburger) [reply components/nordigen/actions/create-requisition-link/create-requisition-link.mjs:7]: Google's Developer docs style guide: Use double quotes for strings.

File: components/nordigen/actions/create-requisition-link/create-requisition-link.mjs [hunk 2]
```diff
@@ -47,9 +47,7 @@ export default {
       institutionId: this.institutionId,
       maxHistoricalDays: this.maxHistoricalDays,
       accessValidForDays: this.accessValidForDays,
-      accessScope: [
-        this.accessScope,
-      ],
+      accessScope: this.accessScope,
```
Comment (dylburger) [inline components/nordigen/actions/create-requisition-link/create-requisition-link.mjs:50]:
```suggestion
      accessScope: this.accessScope,
```

Comment (dylburger) [reply components/nordigen/actions/create-requisition-link/create-requisition-link.mjs:50]: Google's Developer docs style guide: Use double quotes for strings.

File: components/nordigen/nordigen.app.mjs [hunk 3]
```diff
@@ -61,7 +61,7 @@ export default {
       description: "Number of days the user agreement will grant access to when listing transactions.",
     },
     accessScope: {
-      type: "string",
+      type: "string[]",
```
Comment (dylburger) [inline components/nordigen/nordigen.app.mjs:64]:
```suggestion
      type: "string[]",
```

Comment (dylburger) [reply components/nordigen/nordigen.app.mjs:64]: Google's Developer docs style guide: Use double quotes for strings.

**Review by alysonturing — CHANGES_REQUESTED** [review]
@LucBerge  You need to increment the version of 7 component(s). Please see [this output](https://github.com/PipedreamHQ/pipedream/actions/runs/4875680165/jobs/8698192390?pr=6239) and https://pipedream.com/docs/components/guidelines/#versioning for more information.

**Comment by michelle0927** [comment]
Testing passed. Ready for release!

Test report
[Nordigen_6239.pdf](https://github.com/PipedreamHQ/pipedream/files/11400308/Nordigen_6239.pdf)

**Pull request merged**